content/en/docs/contribute/style/style-guide.md

```diff
@@ -185,7 +186,7 @@ Do | Don't
 Set the value of `imagePullPolicy` to Always. | Set the value of `imagePullPolicy` to "Always".
 Set the value of `image` to nginx:1.8. | Set the value of `image` to `nginx:1.8`.
 Set the value of the `replicas` field to 2. | Set the value of the `replicas` field to `2`.
-{{< /table >}} 
+{{< /table >}}
 
 
```

Comment:
Try to do a single sentence per line. It eases up diff reading and helps the localization teams to have a sentence by sentence based translation that is easier to do because it goes lines by lines.